doc/radosgw/placement.rst

```diff
@@ -107,34 +113,32 @@ Adding a Placement Target
 To create a new placement target named ``temporary``, add it to
 the zonegroup:
 
-::
+.. prompt:: bash #
 
-  $ radosgw-admin zonegroup placement add \
-        --rgw-zonegroup default \
-        --placement-id temporary
+   radosgw-admin zonegroup placement add --rgw-zonegroup default \
+                                           --placement-id temporary
 
 Then provide the zone placement info for that target:
 
-::
+.. prompt:: bash #
 
-  $ radosgw-admin zone placement add \
-        --rgw-zone default \
-        --placement-id temporary \
-        --data-pool default.rgw.temporary.data \
-        --index-pool default.rgw.temporary.index \
-        --data-extra-pool default.rgw.temporary.non-ec
+   radosgw-admin zone placement add --rgw-zone default \
+                                      --placement-id temporary \
+                                      --data-pool default.rgw.temporary.data \
+                                      --index-pool default.rgw.temporary.index \
+                                      --data-extra-pool default.rgw.temporary.non-ec
```

Comment:
If the intent is to line up the list of switches vertically, should there be two fewer leading spaces?

Reply:
That is the intention.  
But the prompt is not left of the margin, the left margin for both the first line (with prompt) and continuation lines is at the same position. So the continuation lines need to be indented with 2 more spaces so that the rendered docs line up to the first line that has extra `# `.

Reply:
I suspect there may be a way to "fix" this, since the directive understands continuation lines in the first place (lines ending in `\`) and doesn't put a prompt for such lines. Maybe it could be modified to insert two spaces instead.

I'm too much of a newbie with Sphinx at this point but would love to improve this at some point.  
Right now I cannot think of a scenario where continuation lines would *not* need to begin at the same column as the command after the prompt.

Reply:
At this point I daresay you already know more about Sphinx than I do.  So long as it looks decent when rendered, it's all good.